lib/src/lints/flutter/prefer_dedicated_media_query_method.dart

```diff
@@ -97,6 +119,30 @@ class _ReplaceWithDedicatedMethod extends DartFix {
     AnalysisError analysisError,
     List<AnalysisError> others,
   ) {
+    context.registry.addPrefixedIdentifier((node) {
+      if (!analysisError.sourceRange.intersects(node.sourceRange)) return;
+
+      final typeName =
+          node.prefix.staticType?.getDisplayString(withNullability: false);
+      if (typeName != 'MediaQueryData') return;
```

Comment:
We probably do not need to do a type check here since the above analysisError source range check already did that.